bindata/manifests/operator-webhook/003-webhook.yaml

```diff
@@ -33,7 +33,7 @@ webhooks:
         namespace: {{.Namespace}}
         path: "/validating-custom-resource"
     rules:
-      - operations: [ "CREATE", "UPDATE", "DELETE" ]
```

Comment:
I think we still need the DELETE operation for sriovnetworknodepolicies.

Reply:
There was some error when trying to delete the non-default policy.  I cannot find the root cause, so remote the deletion check. We add this back after fix that issue.